src/libraries/System.Text.Json/src/System/Text/Json/Serialization/IReadBufferState.cs

```diff
@@ -22,5 +22,7 @@ public abstract ValueTask<TReadBufferState> ReadAsync(
         public abstract void Read(TStream utf8Json);
 
         public abstract void Advance(long bytesConsumed);
+
+        public abstract Utf8JsonReader GetReader(JsonReaderState jsonReaderState);
```

Comment:
I don't think the `Stream` case should ever need `public abstract ReadOnlySequence<byte> Bytes { get; }` so ideally that can be removed.

Reply:
It's used by a few `Debug.Assert` calls